docs/source/features/dataset_management.rst

```diff
@@ -62,19 +62,19 @@ There are plenty of benefits of uploading local datasets to TensorBay.
 Two types of datasets can be read from TensorBay:
 
 - Datasets uploaded by yourself as mentioned in :ref:`features/dataset_management:Upload Dataset`.
-- Datasets uplaoded by the community (i.e. the `open datasets`_).
+- Datasets uploaded by the shared `Open Datasets`_ platform.
 
 .. note::
 
    Before reading a dataset uploaded by the community, fork_ it first.
 
 .. note::
 
-   Visit our `Graviti AI Service(GAS)`_ platform to check the dataset details,
+   Visit `my Datasets`_ panel platform to check the available dataset details,
```

Comment:
Follow the correct method:
![image](https://user-images.githubusercontent.com/32992355/115416014-f6a06e00-a229-11eb-9593-3338e0e02e69.png)